apps/console/src/features/server-configurations/forms/analytics-form.tsx

```diff
@@ -0,0 +1,419 @@
+                        ...resolvedInitialValues,
+                        basicAuthPassword: property.value
+                    };
+
+                    break;
+                case ServerConfigurationsConstants.ANALYTICS_HTTP_CONNECTION_TIMEOUT:
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        httpConnectionTimeout: parseInt(property.value)
+                    };
+
+                    break;
+                case ServerConfigurationsConstants.ANALYTICS_HTTP_READ_TIMEOUT:
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        httpReadTimeout: parseInt(property.value)
+                    };
+
+                    break;
+                case ServerConfigurationsConstants.ANALYTICS_HTTP_CONNECTION_REQUEST_TIMEOUT:
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        httpConnectionRequestTimeout: parseInt(property.value)
+                    };
+
+                    break;
+                case ServerConfigurationsConstants.ANALYTICS_HOSTNAME_VERIFICATION:
+                    resolvedInitialValues = {
+                        ...resolvedInitialValues,
+                        hostNameVerfier: property.value
+                    };
+
+                    break;
+
+            }
+        });
+        setInitialConnectorValues(resolvedInitialValues);        
+    }, [ initialValues ]);
+
+    /**
+     * Prepare form values for submitting.
+     *
+     * @param values - Form values.
+     * @returns Sanitized form values.
+     */
+    const getUpdatedConfigurations = (values: AnalyticsFormValuesInterface): AnalyticsAPIRequestInterface => {
+        const data: AnalyticsAPIRequestInterface = {
+            "__secret__adaptive_authentication.elastic.basicAuth.password": values.basicAuthPassword,
+            "adaptive_authentication.elastic.HTTPConnectionRequestTimeout": values.httpConnectionRequestTimeout,
+            "adaptive_authentication.elastic.HTTPConnectionTimeout": values.httpConnectionTimeout,
+            "adaptive_authentication.elastic.HTTPReadTimeout": values.httpReadTimeout,
+            "adaptive_authentication.elastic.basicAuth.enabled": values.basicAuthEnabled,
+            "adaptive_authentication.elastic.basicAuth.username": values.basicAuthUsername,
+            "adaptive_authentication.elastic.hostnameVerfier": values.hostNameVerfier,
+            "adaptive_authentication.elastic.receiver": values.receiver
+        };
+
+        return data;
+    };
+
+    if (!initialConnectorValues) {
+        return null;
+    }
+
+    return (
+        <div className={ "connector-form" }>
+            <Form
+                id={ FORM_ID }
+                initialValues={ initialConnectorValues }
+                onSubmit={ (values: any) => onSubmit(getUpdatedConfigurations(values)) }
```

Comment:
Can't we use FormValues interface instead of any?

Reply:
Fixed with 1d3ef8fd4f615de27060c959ea91a56ca1f3158f